ledger/acctupdates.go

```diff
@@ -387,7 +406,15 @@ func (au *accountUpdates) lookupKv(rnd basics.Round, key string, synchronized bo
 
 		persistedData, err := au.accountsq.lookupKeyValue(key)
 		if persistedData.round == currentDbRound {
-			return persistedData.value, nil
+			if err == nil {
```

Comment:
Pavel recently made a PR to change the error handling on the similar functions that would be good to copy here. 

#4425